tests/lib/json_data.py

```diff
@@ -356,3 +356,10 @@
         }
     }
 ]
+
+mook_data['valid']['facts']= {
```

Comment:
mock?

Reply:
It is existing name, so would be better to change it in a PR as it will concern all files under tests/unit/

Reply:
ack